modules/administration-guide/pages/enabling-readiness-init-containers.adoc

```diff
@@ -0,0 +1,6 @@
+[id="enabling-readiness-init-containers"]
+:navtitle: Enabling readiness init containers
+:keywords: init-container, readiness-init-container, startup
+:page-aliases: .:enabling-readiness-init-containers
+
```

Comment:
```suggestion

{prod-short} Operator installs {prod-short} and starts its containers in the right order. If a node with {prod-short} is restarted and all {prod-short} containers start simultaneously, some containers may fail because some other component they depend on is not ready. To avoid such failures, readiness init containers queue the containers for {prod-short} components to start in the right order. Readiness init containers are not enabled by default.

You can enable readiness init containers according to the installation method used to install {prod-short}:
* xref:readiness-init-containers.adoc#enabling-readiness-init-containers-for-operator-installer_{context}[]
* xref:readiness-init-containers.adoc#enabling-readiness-init-containers-for-olm-installer_{context}[]

To disable enabled readiness init containers, proceed according to the installation method used to install {prod-short}:
* xref:readiness-init-containers.adoc#disabling-readiness-init-containers-for-operator-installer_{context}[]
* xref:readiness-init-containers.adoc#disabling-readiness-init-containers-for-olm-installer_{context}[]

include::partial$proc_enabling-readiness-init-containers-for-operator-installer.adoc[leveloffset=+1]
include::partial$proc_enabling-readiness-init-containers-for-olm-installer.adoc[leveloffset=+1]
include::partial$proc_disabling-readiness-init-containers-for-operator-installer.adoc[leveloffset=+1]
include::partial$proc_disabling-readiness-init-containers-for-olm-installer.adoc[leveloffset=+1]
```